daemon/images/store_test.go

```diff
@@ -75,14 +74,14 @@ func TestImageDelete(t *testing.T) {
 
 		ls, err := images.leases.List(ctx)
 		assert.NilError(t, err)
-		assert.Check(t, cmp.Equal(len(ls), 1), ls)
+		assert.Equal(t, len(ls), 1, ls)
```

Comment:
`assert.Equal(t, a, b)` is equivalent to `assert.Assert(t, is.Equal(a, b))`, which aborts the test on failure. https://github.com/golang/go/wiki/TestComments#keep-going

Reply:
@corhere Oh! Forgot about this one; I recall I started to change it back, and then recalled I looked at this when writing the patch, and considered "would it even make sense to continue here? 

- Here, we get the list of leases, and expect here to be 1 (would be likely `0` of it failed)
- Below we check the list after deleting; that check may _PASS_ if it failed here (if it's `0` above, and still `0` below, that check would incorrectly "PASS", but for the wrong reason.


So from that, I concluded that it would probably be better to fail early; wdyt? (Or was your concern that we wouldn't call the `images.Delete()`?)

Reply:
The title, description and commit message all suggest that this PR is purely a renaming chore, with the implication that no semantic changes are being introduced. Irrespective of whether this change makes sense, I think it belongs in a separate commit.

As far as "would it make sense to continue," [yes, I think it would.](https://github.com/golang/go/wiki/TestComments#keep-going) It doesn't matter if some checks may pass for the wrong reason once the test has been marked as failed. It isn't going to un-fail the test run. Everything that happens after the test is first marked as failed are clues which help to characterize the bug(s) caught by the test. If `images.Delete` returns an error, that's a clue. If it returns `err == nil`, that's _also_ a clue. The list of leases starts at length 2? We can eliminate hypotheses for how that happened by considering how the number of leases changes after `images.Delete()` is called. Aborting the test on the first failed check leaves us with only the failed check to start debugging the test.